examples/charm++/future/fib.C

```diff
@@ -0,0 +1,44 @@
+#include <ck/future.h>
+#include "fib.decl.h"
+
+#define THRESHOLD 3
+
+struct Main : public CBase_Main {
+  Main(CkArgMsg* m) { thisProxy.run(atoi(m->argv[1])); }
```

Comment:
delete m

Reply:
I changed the constructor to a `nokeep` entry method, which should resolve this issue. This was copied verbatim from the existing `fib` example, so we may want to fix that at some point as well.

Reply:
At one point, (and maybe still), we planned on making the Main ctor nokeep by default: https://github.com/UIUC-PPL/charm/pull/2423

Reply:
Not sure if we want to revisit that change